src/packages/@deprecate/assets/crossicon.svelte

```diff
@@ -3,8 +3,8 @@
 </script>
 
 <svg
-  width="15"
-  height="15"
+  width="12"
+  height="12"
```

Comment:
instead of changing the size here, take width and height as prop and set the previous size as default.